cmd/site/templates.go

```
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"html/template"
+	"io"
+
+	"github.com/golang-insiders/site/internal/types"
+)
+
+type Templates struct {
+	templates *template.Template
+}
+
+func newTemplate() Templates {
+	return Templates{
+		templates: template.Must(template.ParseGlob("public/*.html")),
```

templates.Must panics If it is not successful, could we return an error here instead?